scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.html

```diff
@@ -28,8 +28,8 @@
     </div>
 
     <div id='sc-meta-data-panel'>
-        <sc-detailarea (valueChange)="collapsePanel($event)"
-                       [branchInformationTree]="branchInformationTree" [buildInformationTree]="buildInformationTree"></sc-detailarea>
+        <sc-detailarea (togglePannelCollapsedValue)="collapsePanel($event)"
+                       [mainDetailsSections]="mainDetailsSections" key="mainView"></sc-detailarea>
```

Comment:
I think the additional details sections are missing - see additionalDetailsSections in scenarios overview

Reply:
I understand now, here its different because branch and build are displayed in two hi level sections. its good. Sorry.